src/SessionSetup/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2021 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+cmake_minimum_required(VERSION 3.15)
+
+project(SessionSetup)
+add_library(SessionSetup STATIC)
+
+target_compile_options(SessionSetup PRIVATE ${STRICT_COMPILE_FLAGS})
+
+target_sources(
+  SessionSetup
+  PUBLIC  include/SessionSetup/Connections.h
+          include/SessionSetup/ConnectToStadiaWidget.h
+          include/SessionSetup/DeploymentConfigurations.h
+          include/SessionSetup/DoubleClickableLabel.h
+          include/SessionSetup/Error.h
+          include/SessionSetup/OverlayWidget.h
+          include/SessionSetup/ProcessItemModel.h
+          include/SessionSetup/SessionSetupDialog.h
+          include/SessionSetup/ServiceDeployManager.h
+          include/SessionSetup/TargetConfiguration.h
+          include/SessionSetup/TargetLabel.h)
+
+target_include_directories(SessionSetup PUBLIC ${CMAKE_CURRENT_LIST_DIR}/include)
+
+target_sources(
+  SessionSetup
+  PRIVATE ConnectToStadiaWidget.cpp
+          ConnectToStadiaWidget.ui
+          DeploymentConfigurations.cpp
+          Error.cpp
+          OverlayWidget.cpp
+          OverlayWidget.ui
+          ProcessItemModel.cpp
+          SessionSetupDialog.cpp
+          SessionSetupDialog.ui
+          ServiceDeployManager.cpp
+          TargetLabel.cpp
+          TargetLabel.ui
+          ../../icons/orbiticons.qrc)
+
+target_link_libraries(
+  SessionSetup
+  PUBLIC  CaptureFileInfo
+          ClientData
+          ClientServices
+          CONAN_PKG::grpc
+          CONAN_PKG::abseil
+          MetricsUploader
+          OrbitBase
+          OrbitGgp
+          OrbitSsh
+          OrbitSshQt
+          Qt5::Widgets
+          QtUtils)
```

Comment:
Could you put our internal targets at the top and sort them alphabetically?

Reply:
done

Reply:
thx